hermes-common/src/main/java/pl/allegro/tech/hermes/common/metric/Gauges.java

```diff
@@ -10,10 +10,15 @@ public class Gauges {
             EVERYONE_CONFIRMS_BUFFER_AVAILABLE_BYTES = "everyone-confirms-buffer-available-bytes",
             EVERYONE_CONFIRMS_COMPRESSION_RATE = "everyone-confirms-compression-rate-avg",
             EVERYONE_CONFIRMS_FAILED_BATCHES_TOTAL = "everyone-confirms-failed-batches-total",
+            EVERYONE_CONFIRMS_METADATA_AGE = "everyone-confirms-metadata-age",
```

Comment:
ahh, I see that there will be some conflicts 😄 https://github.com/allegro/hermes/pull/1595/files#diff-2591d9b8543f12a041752e0c2e12c7f5d7bd7eab27b4c71dbb81d177df9f7b14L13